js/react_native/onnxruntime-react-native.iphoneos.podspec

```diff
@@ -0,0 +1,27 @@
+require "json"
```

Comment:
Do you need 2 podspec for iphoneos and iphonesimulator?
We are trying to publish a single c/c++ package for ios ort, can you use that as a dependent in the future or can use our ios build script (WIP) to build a unified package?

Reply:
That would be great. I'll try your framework once it's ready.

Reply:
I'll try in the separate PR.